src/scenes/Home/components/MentorApplication/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import { Steps, Button, Row, Col, Input, message } from 'antd';
+import logo from '../../scholarx.png';
+import styles from './styles.css';
+
+const { Step } = Steps;
+const { TextArea } = Input;
+
+function MentorApplication() {
+  const [current, setCurrent] = useState<number>(0);
+  const selectGo = () => {
+    setCurrent(current + 1);
```

Comment:
```suggestion
    setCurrent(2);
```
We know that the next step's number is `2`. Therefore no need to do calculations.